mkscripts/makedll.pl

```diff
@@ -512,6 +497,17 @@
   print OUT "extern bool ecmdDebugOutput;\n";
   print OUT "#endif\n\n\n";
 
+  print OUT "void loadSymbol($ARGV[0]FunctionIndex_t enumname, const char * funcname) {\n";
```

Comment:
would this function be defined in each extension? would we want it static?

Reply:
The way it works now is each *ClientCapiFunc.C has their own copy of it local in the .C.  That handles the different enunname values (`ecmdFunctionIndex_t`, `cipFunctionIndex_t`, etc..)

Reply:
okay, so it is not an extern "C" func, so the arguments will make it a unique function. cool.